src/main.js

```diff
@@ -1,20 +1,28 @@
 // Register service worker + subscribe to push notifications
+const SUBSCRIBE_ENDPOINT = 'https://api.treasurehacks.org/notifications/live-site/subscribe';
 if ('serviceWorker' in navigator) {
   navigator.serviceWorker.register(new URL('/src/service-worker.js', import.meta.url)).then(sw => {
-    sw.pushManager.subscribe({
-      userVisibleOnly: true,
-      // Server's public key
-      applicationServerKey: 'BD_DXIXKbM6eMNm6SejSYnaZbVaM_ekjc_WAXBEPa4pUpf9E7kXEpV-8KtzgSrcl_rk-_yGbonNwyLLe69aJGvE'
-    }).then(sub => {
-      window.sub = sub;
-    });
+    if ('Notification' in window) {
```

Comment:
Same as comment below; guard clauses help with not needing so many nested scopes